samples: crypto: Add EdDSA sample (#12647)
nrfconnect/sdk-nrf: samples/crypto/ecdsa/README.rst

1. Initialization:

   a. The Platform Security Architecture (PSA) API is initialized.

>    # A random Elliptic Curve Cryptography (ECC) key pair is generated in the PSA crypto keystore for signing usage.
>    # The public key of the ECC key pair is exported to the application.
>    # The exported public key is imported in the PSA crypto key storage for verify usage.

Reviewer: ```suggestion
   #. A random Elliptic Curve Cryptography (ECC) key pair is generated in the PSA crypto keystore for signing purposes.
   #. The public key of the ECC key pair is exported to the application.
   #. The exported public key is imported into the PSA crypto key storage to be used for verification.
```

Author: usage is the PSA specification word used.
https://armmbed.github.io/mbed-crypto/html/api/keys/policy.html#key-usage-flags

Reviewer: Glancing through it quickly, it seems usage is mentioned when talking about usage flags.
Should these sentences rather mention usage flags instead then?
E.g.: A random Elliptic Curve Cryptography (ECC) key pair is generated in the PSA crypto keystore with the signing usage flag.
Or verfication usage flag for that later sentence.
But if not mentioning the usage flags, it does not seem to me like usage is needed in the sentence.
